src/app/view/dynamic-html.js

```diff
@@ -0,0 +1,44 @@
+import {noView, ViewCompiler, ViewSlot, Container, ViewResources, bindable} from 'aurelia-framework';
+
+@noView()
+export class DynamicHtmlCustomElement {
```

Comment:
This component is used to render html dynamically with aurelia's template engine. The reason for this is that the React components are rendered via the aurelia template engine and loaded via the aurelia loader. So just rendering html would not render the React components. Therefore I introduced this custom element which takes the html to render and throw it into the template engine. The generated html is the final version we just append to the DOM.